s3_demo.py

```diff
@@ -0,0 +1,13 @@
+import boto3
+
+def list_files(bucket):
+    """
+    Function to list files in a given S3 bucket
+    """
+    s3 = boto3.client('s3')

```

Comment:
```
s3 = boto3.client('s3')
```
should be placed in the init function in the app.py and then referred wherever needed